falco.yaml

```diff
@@ -182,7 +182,7 @@ http_output:
 # grpc:
 #   enabled: true
 #   bind_address: "0.0.0.0:5060"
-#   threadiness: 8
+#   threadiness: 0
```

Comment:
0 Will automatically figure out the number of cores dynamically

Reply:
Putting the comment also here, thanks